<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageStageWgtClass</class>
 <widget class="QWidget" name="PageStageWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Manufacturer:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Part number:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="varlengthLineEdit" name="partnumberEdit">
     <property name="maxLength">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Lead screw pitch:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Units:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="varlengthLineEdit" name="unitsEdit">
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Max. speed:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="FloatSpinBox" name="maxSpeedSpin">
     <property name="suffix">
      <string> units/s</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Travel range:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="FloatSpinBox" name="travelRangeSpin">
     <property name="suffix">
      <string> units</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Supply voltage:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Max. current consumption:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="FloatSpinBox" name="maxCurrentConsumptionSpin">
     <property name="suffix">
      <string> A</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Horizontal load capacity:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="FloatSpinBox" name="horizontalLoadCapacitySpin">
     <property name="suffix">
      <string> kg</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Vertical load capacity:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="FloatSpinBox" name="verticalLoadCapacitySpin">
     <property name="suffix">
      <string> kg</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="FloatSpinBox" name="leadScrewPitchSpin">
     <property name="suffix">
      <string> units</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="varlengthLineEdit" name="manufacturerEdit">
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="FloatSpinBox" name="supplyVoltageMinSpin">
        <property name="suffix">
         <string> V</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string> - </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FloatSpinBox" name="supplyVoltageMaxSpin">
        <property name="suffix">
         <string> V</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>manufacturerEdit</tabstop>
  <tabstop>partnumberEdit</tabstop>
  <tabstop>leadScrewPitchSpin</tabstop>
  <tabstop>unitsEdit</tabstop>
  <tabstop>maxSpeedSpin</tabstop>
  <tabstop>travelRangeSpin</tabstop>
  <tabstop>supplyVoltageMinSpin</tabstop>
  <tabstop>supplyVoltageMaxSpin</tabstop>
  <tabstop>maxCurrentConsumptionSpin</tabstop>
  <tabstop>horizontalLoadCapacitySpin</tabstop>
  <tabstop>verticalLoadCapacitySpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
